docs/hugo/content/guide/authentication/credential-format.md

```diff
@@ -1,35 +1,45 @@
 ---
-title: Authentication in Azure Service Operator v2
-linktitle: Authentication
+title: Credential Format
+linktitle: Credential Format
 ---
 
```

Comment:
Suggest you need an introductory paragraph here to explain what's found below.

Reply:
Done. I didn't wanna fully repeat what was said on the index page but I did a summary of it.